nusight2/typings/bindings/index.d.ts

```diff
@@ -1,7 +1,11 @@
 declare module 'bindings' {
-  const bindings: (name: string | {
-    bindings: string,
-    module_root: string,
-  }) => any
+  const bindings: (
```

Comment:
What has changed to our formatter??? Are we using the correct formatter config for nusight files?

Reply:
god damnit. I merged in master again to a PR onto another PR...
These formatting changes are on master but not your branch.

The formatter fixes these if we actually run it properly, but doesn't notice if they're not fixed? idk. it's weird. Basically I found these things after I rebuilt target generic with nodejs inside docker. These are old news :shrug: @JosephusPaye